chapter2/linear-elasticity-intro.md

```diff
@@ -0,0 +1,137 @@
+# Linear Elasticity
+
+## The PDE problem
+The governing equations for small elastic deformations of a body $\Omega$ can be expressed as:
+
+$$
+\begin{aligned}
+    -\nabla \cdot \sigma(u) &= f & \text{in } \Omega \\
+    \sigma(u) &= C : \epsilon(u) & \text{in } \Omega \\
+\end{aligned}
+$$
+
+where : 
+ - $\Omega$ is the domain of interest,
+ - $u$ is the displacement vector field,
+ - $\sigma(u)$ is a second-order symmetric tensor, called stress tensor, 
+ - $f$ represents the load vector field (force per unit volume), 
+ <!-- $\kappa$ and $\mu$ are Lamé's elasticity parameters for the material, $I$ denotes the identity tensor,  -->
+ - $\epsilon(u)$ is a second-order symmetric tensor, called strain tensor and by definition $\epsilon(u) := \frac{1}{2}(\nabla u + (\nabla u)^T)$.
+ - $C$ is the elasticity tensor, which relates stress and strain. 
+
+Then, the strong formulation of linear elasticity is :
+Find $u \in V$ such that
+
+$$
+\begin{aligned}
+    -\nabla \cdot \sigma (u) &= f & \text{in } & \Omega \\
+    u &= 0 & \text{on } & \partial \Omega_D \\
+    \sigma(u) \cdot n &= g_T & \text{on } & \partial \Omega_T
+\end{aligned}
+$$
+
+where : 
+- $V = \{ v \in (H^1(\Omega))^3 : v = 0 \text{ on } \partial \Omega_D \}$,
+
+- $g_T$ is the traction vector on the part $\partial \Omega_T$ of the boundary,
+
+- $n$ is the outward normal vector on the boundary $\partial \Omega$,
+
+- $\partial \Omega_D \cap \partial \Omega_T = \emptyset$ and $\partial \Omega = \partial \Omega_D \cup \partial \Omega_T$. 
+- $\partial \Omega_D$ is the Dirichlet boundary condition where the displacement is fixed to zero, and $\partial \Omega_T$ is the Neumann boundary condition where the traction is prescribed.
+
+## The Variational Formulation
+The variational formulation of the linear elasticity equations involves forming the inner product of the PDE with a vector test function $ v \in V $ and integrating over the domain $ \Omega $. This yields:
```

Comment:
Empty spaces next to the `$` delimiters stop GitHub from formatting inline math.
```suggestion
The variational formulation of the linear elasticity equations involves forming the inner product of the PDE with a vector test function $v \in V$ and integrating over the domain $\Omega$. This yields:
```